conftest.py

```diff
@@ -27,11 +25,12 @@ def pytest_configure(config):
 
 
 def pytest_unconfigure(config):
-    db.test.drop_tables(models)
-    db.test.close()
+    db.database.drop_tables(models)
```

Comment:
Don't you need to restore the initial prefix too? Or is that anyway tight to the connection that you close below?